scripts/stripnulls.sh

```diff
@@ -7,7 +7,7 @@
 SED=$(command -v gsed)
 SED=${SED:-"sed"}
 
-FILES=$(grep -rl '"schemaVersion": 3[3456]' devenv)
+FILES=$(grep -rl '"schemaVersion": 3[3456789]' devenv)
```

Comment:
I assume this was updated because you needed to run it...  I *thought* we updated our exports so it strips nulls 🤔 

something to keep an eye on -- if this is still required, we need to fix the export on javascript side

Reply:
yeah, i thought we did too 😢 but indeed, i had to run it in order to remove some nulls. So, clearly, not yet.